arangod/Transaction/Methods.cpp

```diff
@@ -1901,9 +1907,6 @@ OperationResult transaction::Methods::modifyLocal(
                     << (*followers)[i] << " for shard " << collectionName;
                   THROW_ARANGO_EXCEPTION(TRI_ERROR_CLUSTER_COULD_NOT_DROP_FOLLOWER);
                 }
-                LOG_TOPIC(ERR, Logger::REPLICATION)
```

Comment:
Why is this LOG removed. That is IMHO a useful information because it means we need replication to catch up and see what is going on.
Maybe ERROR level is wrong, as it is expected somehow on high load / bad network nevertheless we should still see it.

Reply:
ok you are right this is already logged ;(